server/models/service-process.js

```diff
@@ -128,6 +128,23 @@ module.exports = function extendServiceProcess(ServiceProcess) {
   }
   ServiceProcess.recordProfilingState = recordProfilingState;
 
+  function recordStatusWdUpdate(instanceId, pInfo, callback) {
+    function updateProcessStatus(proc, asyncCb) {
+      proc.isTracing = pInfo.isTracing || false;
```

Comment:
`proc.isTracing = !!pInfo.isTracing` would make more sense to me if you want to coerce to a boolean, but I'd hope that loopback would coerce, since field type in model is boolean, in which case a simple assignment is sufficient.